extension/JuridicalPersion.StructureDefinition.json

```diff
@@ -0,0 +1,53 @@
+{
+  "resourceType": "StructureDefinition",
+  "url": "http://bbmri.de/fhir/StructureDefinition/JuridicalPersion",
+  "name": "JuridicalPersion",
+  "status": "draft",
+  "fhirVersion": "4.0.0",
+  "kind": "complex-type",
+  "abstract": false,
+  "context": [
+    {
+      "type": "element",
+      "expression": "Organization"
+    }
+  ],
+  "type": "Extension",
+  "baseDefinition": "http://hl7.org/fhir/StructureDefinition/Extension",
+  "derivation": "constraint",
+  "differential": {
+    "element": [
+      {
+        "id": "Extension",
+        "path": "Extension",
+        "short": "Optional Extensions Element",
+        "definition": "Optional Extension Element - found in all resources.",
+        "constraint": [
+          {
+            "key": "ext-1",
+            "severity": "error",
+            "human": "Must have either extensions or value[x], not both",
+            "expression": "extension.exists() != value.exists()",
+            "xpath": "exists(f:extension)!=exists(f:*[starts-with(local-name(.), 'value')])"
+          }
+        ],
+        "mapping": [
+          {
+            "identity": "rim",
+            "map": "N/A"
+          }
+        ]
+      },
```

Comment:
In other extensions, I see a fixed value for the Extension.url:

```json
{
                "fixedUri": "http://hl7.org/fhir/StructureDefinition/rendering-markdown",
                "id": "Extension.url",
                "path": "Extension.url",
                "type": [
                    {
                        "code": "uri"
                    }
                ]
            }
```

Reply:
I added these to the extensions for biobank. Although i'm not sure if they are really nescessary since "The definition SHALL be a URI for the Structure Definition defining the extension" is specified on https://www.hl7.org/fhir/extensibility-definitions.html#Extension.url . Forge does not offer an option to set this manually. Still, propably doesn't hurt :)